testing/ostest/ostest_main.c

```diff
@@ -706,6 +719,9 @@ int main(int argc, FAR char **argv)
     }
 
   printf("ostest_main: Exiting with status %d\n", ostest_result);
+#else
+  user_main(1, NULL);
```

Comment:
why need #else

Reply:
test task_create function in flat mode, we can skip in kernel mode
https://github.com/apache/nuttx-apps/blob/master/testing/ostest/ostest_main.c#L681